extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -94,6 +119,40 @@ class AmpYoutube extends AMP.BaseElement {
   }
 
   /** @private */
+  getPlayerReadyPromise_() {
+    return new Promise(resolve => {
```

Comment:
This works with a large enough delay before calling the resolver. 

The reason I had it stubbed out originally is that I couldn't get YouTube messages to work in tests. This was due to using `this.getWin()` instead of global `window` to `addEventListener('message')`.

if using `window` is ok then I don't need to stub this.